CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v58.0.0...main)
 
+* Rust
+  * BREAKING CHANGE: Timing distribution and Custom distribution traits now expose `accumulate_single_sample`. Consumers that make use of the trait will need to implement the new functions ([Bug 1881297](https://bugzilla.mozilla.org/show_bug.cgi?id=1881297))
```

Comment:
The only consumer that should be implementing this trait is the RLB, right?

Reply:
The last time we did this, it came with this message. If it's not a concern, I'll gladly remove it :D